src/scripts/extensions/userDataBoundaryHelper.ts

```diff
@@ -0,0 +1,65 @@
+import { Clipper } from "../clipperUI/frontEndGlobals";
+import { Constants } from "../constants";
+import { HttpWithRetries } from "../http/HttpWithRetries";
+import { UrlUtils } from "../urlUtils";
+import { UserInfoData, AuthType } from "../userInfo";
+import { DataBoundary } from "./DataBoundary";
+
+export class UserDataBoundaryHelper {
```

Comment:
why do we want to make a class for it and not just utilities functions? it doesn't have any class members, so IMO it is fine to just add the functions.

Reply:
Used class implementation for handling any future extensions to the class to add any more functions.